src/main/kotlin/onestep/racingcar/model/Racing.kt

```diff
@@ -11,7 +11,12 @@ class Racing(
             cars.map(Car::run)
             show(cars)
         }
-        return cars
+        return findWinner(cars)
+    }
+
+    private fun findWinner(cars: List<Car>): List<Car> {
+        val list = cars.sortedByDescending { it.position }
+        return if (list.isEmpty()) list else list.filter { it.position == list[0].position }
```

Comment:
list가 empty일 때는 filter로 넘긴 고차함수가 실행되지 않을테니 isEmpty() 체크를 해줄 필요가 없을 것 같아요.
```suggestion
        return list.filter { it.position == list[0].position }
```

그리고 고차함수내에서 바깥의 list를 참조하는 것보다는 다른 방법을 찾아보는 것도 좋겠어요.
지금과 같은 경우는 `cars.sortedByDescending()` 을 호출해주면서 [방어적 복사](https://johngrib.github.io/wiki/defensive-copy.md/)가 일어났기 때문에 문제가 없습니다.
하지만 **방어적 복사를 하지 않은 상태**에서 **멀티 쓰레드 환경에서 다른 곳에서 cars를 변경**했다면 문제가 발생할 수 있습니다.

이번 코드에서는 방어적 복사도 하셨고 외부에서 MutableList를 사용하지도 않으셨네요 👍 👍